eos_data_distribution/SimpleStore/__init__.py

```diff
@@ -40,9 +41,9 @@ def __init__(self, base=None, prefix='/',
 
         if base: self.publish_all_names(base)
 
-    def _path_to_name(self, filename):
-        assert filename.startswith(self.base)
-        file_path = filename[len(self.base):].lstrip('/')
+    def _path_to_name(self, filename, basedir):
+        assert filename.startswith(basedir)
```

Comment:
I’m not so sure. If the base path contains a symlink somewhere, all the paths produced by `walk()` will do as well (I believe), so I don’t think we want to call `realpath()` in `_path_to_name()` or the prefix assertion will fail.